src/Padawan/Domain/Project/Node/TypeHint.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Padawan\Domain\Project\Node;
+
+class TypeHint extends Variable {
+    public $startLine = 0;
+
+    public static function create(Variable $variable, $startLine)
```

Comment:
I mean, there shouldn't be class TypeHint, we can store all this info in variable